src/components/Chat/BottomSheetGif.js

```diff
@@ -109,6 +121,7 @@ const BottomSheetGif = React.forwardRef((props, ref) => {
         refreshing={isLoading}
         onRefresh={onLoad}
         numColumns={2}
+        ListFooterComponent={<View style={{height: 100}} />}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a `ListFooterComponent` with a height of 100 provides some space at the end of the list, which can improve the user experience by preventing the last item from being obscured by any fixed elements at the bottom of the screen.
```diff
+ ListFooterComponent={<View style={{height: 100}} />}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->